vendor/github.com/ServiceComb/go-archaius/.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: go
```

Comment:
I don't think file is needed.

Reply:
@m1093782566 , we can remove the yaml file but a dep ensure will create it again..Please let me know whether this should be removed.

Reply:
Okay, just keep this file.

Reply:
Updated